rubygem-puppetdb_foreman/rubygem-puppetdb_foreman.spec

```diff
@@ -83,6 +83,9 @@ GEMFILE
 %exclude %{gem_dir}/cache/%{gem_name}-%{version}.gem
 
 %changelog
+* Sun Oct 25 2015 Daniel Lobato Garcia <elobatocs@gmail.com> 0.1.3-1.fm1_10
```

Comment:
Can you remove this please?  tito generates changelogs automatically, and the version number's incorrect anyway.